package.json

```diff
@@ -1,34 +1,65 @@
 {
-  "name": "hubspot-local-dev-tools",
-  "version": "1.0.0",
-  "description": "Local development tools for working with HubSpot",
+  "name": "@hubspot/cli",
+  "version": "6.2.0",
+  "description": "CLI for working with HubSpot",
```

Comment:
This might be a pending conflict, but I think we changed this to `The official CLI for developing on HubSpot`